dockstore-integration-testing/src/test/java/io/dockstore/webservice/TokenResourceIT.java

```diff
@@ -90,7 +90,7 @@
 @ExtendWith(TestStatus.class)
 @ExtendWith(HoverflyExtension.class)
 @HoverflyCore(mode = HoverflyMode.SIMULATE)
-@Tag(NonConfidentialTest.NAME)
+@Tag(HoverflyTest.NAME)
```

Comment:
Similar question for explanation, this one is a tag switch though rather than adding